src/controls/cGameControlsContext.h

```diff
@@ -62,6 +62,9 @@ class cGameControlsContext : public cInputObserver, cScenarioObserver {
     bool shouldDrawToolTip() const {
         return m_drawToolTip;
     }
+    bool isMouseRightButtonPressed() const{
```

Comment:
preferably this goes via `event`s , why need this?

Reply:
I create it for future to implement drawToolTip that is broken because i didn't have key access yet on this part of code. I wasn't sure how I could then link it to events.